pkg/provider/provider.go

```diff
@@ -136,3 +137,13 @@ func MergeDeploymentVars(ms ...map[string]string) map[string]string {
 	}
 	return res
 }
+
+// GetDefaultDeploymentVars shows default deployment variables.
+func (dr *DeploymentResource) GetDefaultDeploymentVars(parseContext *kingpin.ParseContext) error {
+	fmt.Print("-------------------\n   DeploymentVars   \n------------------- \n")
+	for key, value := range dr.FlagDeploymentVars {
```

Comment:
I don't see we have this method set for `DeploymentResource`, this method made more sense in the provider specific files that you added in your previous commit. It will also have sort of a clean separation between `FlagDeploymentVars`, `DefaultDeploymentVars` and `providerDeploymentVars`

Why do you think  we should have this here? :thinking: